app/tests/spec/views/sign_in_recovery_code.js

```diff
@@ -84,6 +84,7 @@ describe('views/sign_in_recovery_code', () => {
     it('renders the view', () => {
       assert.lengthOf(view.$('#fxa-recovery-code-header'), 1);
       assert.include(view.$('.verification-recovery-code-message').text(), 'recovery code');
+      assert.equal(view.$('#use-backup-link').attr('href'), '/signin_totp_code');
```

Comment:
This tests for the links that were already there, do you want a test for the new link?

Reply:
Updated!